app/views/admin/attachments/_attachments.html.erb

```diff
@@ -13,6 +13,9 @@
     {
       text: "Title"
     },
+    {
+      text: "Type"
+    },
     {
       text: "File"
```

Comment:
I think the "File" column also needs renaming to "Attachment"

Reply:
Sorry missed that in the doc